ui/imports/shared/popups/ProfilePopup.qml

```diff
@@ -288,12 +290,12 @@ StatusModal {
         },
 
         StatusButton {
-            property bool isAdded: RootStore.contactsModuleModel.isAdded(fromAuthor)
+            property bool isAdded: Utils.getContactDetailsAsJson(userPublicKey).isContact
```

Comment:
Might be worth adding a `isContact` property above since we use that property a lot in this file

Reply:
@jrainville yes of course, make sense. Done.